Tests/SwiftParserTest/translated/ErrorsTests.swift

```diff
@@ -87,7 +87,52 @@ final class ErrorsTests: XCTestCase {
       """,
       diagnostics: [
         DiagnosticSpec(locationMarker: "1️⃣", message: "expected expression after ternary operator", fixIts: ["insert expression"])
-      ]
+      ],
+      fixedSource: """
+        func one() {
```

Comment:
Oh, this is a huge test. It would be nice if we could split it up into multiple `assertParse` calls because really only the first requires `fixedSource`.

Reply:
Splited each `do` into their own assert. 
Also removed the wrapping function as I didn't see any value. 

Wanted to remove code that is not needed.